admin/coupon_admin.php

```diff
@@ -318,6 +317,17 @@
         ];
         zen_db_perform(TABLE_COUPONS_DESCRIPTION, $sql_data_desc_array, 'update', "coupon_id = " . (int)$_GET['cid'] . " and language_id = " . (int)$languages[$i]['id']);
       }
+      // referrers
+      $trimmed_referrers = array_map(fn ($referrer) => trim($referrer), explode(',', $_POST['coupon_referrer'] ?? []));
```

Comment:
PSR-12, 4.7: "...  there MUST NOT be a space between the method or function name and the opening parenthesis ..."

Reply:
There aren't any.

Reply:
Oh you meant between the `fn` keyword and the function parameter.